src/services/info/street-view.ts

```diff
@@ -0,0 +1,15 @@
+export async function loadGoogleapis() {
+  return await new Promise((resolve /*, reject*/) => {
```

Comment:
are we going to load google api the old way? There is a npm for google map api:
https://developers.google.com/maps/documentation/javascript/load-maps-js-api#js-api-loader
(did not test)

Reply:
ah, that looks interesting, but then the google API would get bundled in the library, instead of being loaded on demand ?